packages/mc-html-template/package.json

```diff
@@ -69,8 +69,7 @@
       [
         "@babel/plugin-transform-runtime",
         {
-          "corejs": 3,
-          "helpers": false
```

Comment:
It's `true` by default: https://babeljs.io/docs/en/babel-plugin-transform-runtime#helpers